jerry-core/vm/opcodes.h

```diff
@@ -54,6 +54,19 @@ typedef enum
   NUMBER_BITWISE_NOT, /**< bitwise NOT calculation */
 } number_bitwise_logic_op;
 
+#if ENABLED (JERRY_ESNEXT)
+
+/**
+ * Types for opfunc_create_executable_object.
+ */
+typedef enum
+{
+  CREATE_EXECUTABLE_OBJECT_GENERATOR, /**< create a generator function */
+  CREATE_EXECUTABLE_OBJECT_ASYNC, /**< create an async function */
+} create_executable_object_type_t;
```

Comment:
How about `ecma_executable_object_type_t` instead? The `create` prefix is quite strange for me.

Reply:
It is the name of the function which uses it. It is not an object type. ecma_create_executable_object_type_t could be, but that is quite long